src/synapse/azext_synapse/_help.py

```diff
@@ -154,122 +163,165 @@
 
 helps['synapse sql pool resume'] = """
 type: command
 short-summary: Resume a SQL pool.
-long-summary: Resume a SQL pool with SQL pool name.
 examples:
   - name: Resume SQL pool.
     text: |-
-        az synapse sql pool resume --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool resume --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool delete'] = """
 type: command
 short-summary: Delete a SQL pool.
-long-summary: Delete a SQL pool with SQL pool name.
 examples:
   - name: Delete SQL pool.
     text: |-
-        az synapse sql pool delete --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool delete --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool wait'] = """
 type: command
 short-summary: Place the CLI in a waiting state until a condition of a SQL pool is met.
 """
 
+helps['synapse workspace firewall-rule'] = """
+type: group
+short-summary:  Manage a workspace's firewall rules.
+"""
+
+helps['synapse workspace firewall-rule create'] = """
+type: command
+short-summary: Create a firewall Rule.
+examples:
+  - name: Create a firewall Rule.
+    text: |-
+        az synapse workspace firewall-rule create --name allowAll --workspace-name testsynapseworkspace \\
+        --resource-group rg --start-ip-address 0.0.0.0 --end-ip-address 255.255.255.255
+"""
+
+helps['synapse workspace firewall-rule show'] = """
+type: command
+short-summary: Get a firewall rule.
+examples:
+  - name: Get a firewall rule.
+    text: |-
+        az synapse workspace firewall-rule show --name rule1 --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse workspace firewall-rule list'] = """
+type: command
+short-summary: List all firewall rules.
+examples:
+  - name: List all firewall rules.
+    text: |-
+        az synapse workspace firewall-rule list --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse workspace firewall-rule delete'] = """
+type: command
+short-summary: Delete a firewall rule.
+examples:
+  - name: Delete a firewall rule.
+    text: |-
+        az synapse workspace firewall-rule delete --name rule1 --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse workspace firewall-rule wait'] = """
+type: command
+short-summary: Place the CLI in a waiting state until a condition of a firewall rule is met.
+"""
+
 helps['synapse spark batch'] = """
 type: group
 short-summary: Manage Synapse Spark batch jobs.
 """
 
 helps['synapse spark batch create'] = """
 type: command
-short-summary: Submit spark batch job.
-long-summary: Submit a spark batch job to a specific spark pool.
+short-summary: Submit Spark batch job.
 examples:
-  - name: Submit a java spark batch job to a specific spark pool.
+  - name: Submit a java Spark batch job.

```

Comment:
>java [](start = 19, length = 4)

Java

Reply:
fixed

---
In reply to: [418040789](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418040789) [](ancestors = 418040789)